client/os.cpp

```diff
@@ -47,9 +47,15 @@ namespace tec {
 
 		// Create a windowed mode window and its OpenGL context.
 		glfwWindowHint(GLFW_OPENGL_FORWARD_COMPAT, GL_TRUE);
-		glfwWindowHint(GLFW_CONTEXT_VERSION_MAJOR, glMajor);
-		glfwWindowHint(GLFW_CONTEXT_VERSION_MINOR, glMinor);
-		//glfwWindowHint(GLFW_OPENGL_PROFILE, GLFW_OPENGL_CORE_PROFILE); // Only with GL 3.2
+    #ifdef __APPLE__
+		glfwWindowHint(GLFW_OPENGL_PROFILE, GLFW_OPENGL_CORE_PROFILE); // Apple requires Core for modern GL
```

Comment:
If we add a check to ensure that MAJOR and MINOR are at least 3.2, CORE_PROFILE can be flag for all OSes.

Reply:
So you want to automatically enable Core on *any* platform if the requested version is >= 3.2?